operations/project-health-review.md

```diff
@@ -0,0 +1,63 @@
+- **Long-term planning exists.**
+ - What to expect: The project defines long term goals, objectives, features, or releases. It is not expected the project will achieve every defined goal or objective within the timeframe or release they specify, rather that they are considering the future of the project and tracking completion of those activities. This may be extremely lightweight in the form of identifying primary components in a v1.0 release, or it may be more detailed and use project boards, milestones, and other planning flows.
+ - Verify: Look for the presence of an up-to-date roadmap file (determined by the timeframe the roadmap represents), project goals, project boards, etc
+- **Development is ongoing and progressive towards versioned releases.**
+ - What to expect: Early projects are likely to have significant variability in the frequency by which development happens. They may experience bursts of contributions and commits, slow steady pacing, or a combination thereof. Each project is different, however we should see some form of progress which may be features and bug fixes, stability improvements, project packaging infrastructure, and tests. Projects should not appear to be in maintenance or sustainment mode - this is an indicator of a project health concern.
+ - Verify: Review the Overall-project-statistics-table on https://devstats.cncf.io/ for the project, if last year’s overall project statistics table is available, compare. Look to see if a versioned release is present. Additionally, look to see if they are following semantic versioning, are creating new tags on major feature development and there is evidence of past stable released versions (semantic versioning is a more mature requirements so verify the project's level).
+- **Adoption is picking up.**
+ - What to expect: While the project is early, we may see one or two more adopters, users, or entities exploring the viability of the project for their needs. We’re not likely to see significant changes within their adoption file, however indicators of features requests and indicators of positive attention by potential adopters. More mature projects should have increasing adopters and these may be recorded in the ADOPTERS.md file or equivalent.
+ - Verify: Look for changes to the adopter file, organization/company blogs, articles and talks concerning the project, look for issues marked as feature or other labels from non-maintainers. Determine if existing adopters are continuing to engage.
+- **Self-awareness.**
+ - What to expect: The project has reasonable awareness of their overall maturity and progress to towards the next level (or maintaining their graduation). They understand their path to the next level and any activities still outstanding in order to be ready to apply. If the project is a graduated project, they are aware of the necessary activities in maintaining good standing with their adopters and community.
```

Comment:
would be good to provide a link for the `necessary activities`.  I'd be favor of annual report from graduated project.